src/Algebra/Morphism/GroupMonomorphism.agda

```diff
@@ -0,0 +1,76 @@
+{-# OPTIONS --without-K --safe #-}
```

Comment:
You're missing a header here. Look at any other file in the library
and grab & modify the top banner.